src/storage/upgrade_project.ts

```diff
@@ -45,8 +45,16 @@ export async function upgradeProjectIfNecessary(
       // @ts-ignore
       case '0.0.1':
         upgradeFrom_001_to_002(storage, projectName, projectInfo);
+      // Intentional fallthrough
+      // @ts-ignore
       case '0.0.2':
         upgradeFrom_002_to_003(storage, projectName, projectInfo);
+      // Intentional fallthrough
+      // @ts-ignore        
+      case '0.0.3':
+        upgradeFrom_003_to_004(storage, projectName, projectInfo);
+      default:
```

Comment:
Why is this default case here? There's no break before it so, after my project upgrades from 003 to 004, it falls through to here and throws the error.